packages/@aws-cdk/pipelines/lib/lambda/index.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+import json
+import time
+
+cp = boto3.client('codepipeline')
+
+def handler(event, context):
+  time.sleep(10) # wait 10 seconds for manual approval action to run
```

Comment:
Fixed sleep delays are almost never a good idea.

* What if 10 seconds isn't enough, for whatever reason? We fail.
* What if 1 second was enough, we 're now wasting 9 seconds.

A typical way to write this pattern is something like:

```py
deadline = now() + 5 minutes # or whatever
while now() < deadline:
  if try_to_do_action():
    break
  sleep(1)
else:
  oh_no_we_failed()
```